src/components/Header/Header.tsx

```diff
@@ -1,54 +1,61 @@
 import React, { useState } from "react";
 // import { Header, Image } from 'semantic-ui-react'
-import Curiologo from "../assets/images/Csmall.png";
-import "./header.css";
+import Curiologo from '../assets/images/Csmall.png'
+import './header.css';
+import { Link ,BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
```suggestion
import { Link ,BrowserRouter as Router } from 'react-router-dom';
```

Route & Switch are imported but not used.